model/bitwarden/organization.go

```diff
@@ -0,0 +1,108 @@
+package bitwarden
+
+import (
+	"errors"
+
+	"github.com/cozy/cozy-stack/model/bitwarden/settings"
+	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/crypto"
+	"github.com/cozy/cozy-stack/pkg/metadata"
+)
+
+// OrgMemberStatus is a type for the status of an organization member
+type OrgMemberStatus int
+
+const (
+	// OrgMemberInvited is used when the member is invited but has not yet
+	// accepted the invitation.
+	OrgMemberInvited = 0
+	// OrgMemberAccepted is used when the member is accepted but the owner has
+	// not yet confirmed that the fingerprint is OK.
+	OrgMemberAccepted = 1
+	// OrgMemberConfirmed is used when the member is confirmed, and has access
+	// to the organization key to decrypt/encrypt ciphers.
+	OrgMemberConfirmed = 2
```

Comment:
What if the user reject the invitation? Is this a new status? Or is the member removed from the organisation?

Reply:
I don't know yet, but my guess would be that the member is removed from the organization.